ReactWindows/ReactNative.sln

```diff
@@ -11,22 +10,36 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ReactNative.Tests", "ReactN
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ChakraBridge", "ChakraBridge\ChakraBridge.vcxproj", "{4B72C796-16D5-4E3A-81C0-3E36F531E578}"
 EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "ReactNative.Shared", "ReactNative.Shared\ReactNative.Shared.shproj", "{0939B6FE-D05D-4663-BC24-A68CA9324BB4}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ReactNative.Shared.Tests", "ReactNative.Shared.Tests\ReactNative.Shared.Tests.csproj", "{B4EEA2E5-4164-4043-B8A8-8CBD7F205414}"
+EndProject
 Global
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		ReactNative.Shared\ReactNative.Shared.projitems*{0939b6fe-d05d-4663-bc24-a68ca9324bb4}*SharedItemsImports = 13
+		ReactNative.Shared\ReactNative.Shared.projitems*{b4eea2e5-4164-4043-b8a8-8cbd7f205414}*SharedItemsImports = 4
+		ReactNative.Shared\ReactNative.Shared.projitems*{c7673ad5-e3aa-468c-a5fd-fa38154e205c}*SharedItemsImports = 4
+	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|ARM = Debug|ARM
```

Comment:
The problem with adding back Any CPU to the solution file is that people don't think to change the Platform setting when first trying to deploy the application.  I noticed that the ReactNative.Shared.Tests project is only Any CPU, but I wonder if it is possible to still remove the solution level configuration for Any CPU (since you've mapped all other platforms anyway).